angular/src/lib/stepper/stepper.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, OnInit, Input, Output, EventEmitter, AfterViewInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'md-stepper',
+  templateUrl: './stepper.component.html',
+  styleUrls: ['./stepper.component.scss'],
+})
+export class StepperComponent implements OnInit, AfterViewInit {
+  @Input() public steps: {
```

Comment:
Missing @prop notations.  @daanishnasir can help clarify the style needed.

Reply:
Done